.circleci/config.yml

```diff
@@ -69,19 +69,7 @@ jobs:
     - run:
         command: |-
           make tools
-
```

Comment:
The changes here don't seem correct. This file is generated by running `make ci-config` in the `.circleci` directory. It is build from the various files in `.circleci/config/`

Reply:
Ah, thanks for the heads up, I manually changed this, I will run the generation instead, thanks!

Reply:
Addressed in https://github.com/hashicorp/boundary/pull/1944/commits/2de86ba9f3ac57e824bf42d3575a62996a5a68e7